schemas/mw-page-edit.yaml

```diff
@@ -0,0 +1,70 @@
+title: MediaWiki Page Edit
+description: Represents a MW Page Edit event
+type: object
+properties:
+  # global event fields
+  meta:
+    type: object
+    properties:
+      uri:
+        type: string
+        format: uri
+        description: the unique URI identifying the event
+      request_id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique UUID v1 ID of the event derived from the X-Request-Id header
+      id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique ID of this event; should match the dt field
+      dt:
+        type: string
+        format: date-time
+        description: the time stamp of the event, in ISO8601 format
+      domain:
+        type: string
+        description: the domain the event pertains to
+    required:
+      - uri
+      - request_id
+      - id
+      - dt
+      - domain
+  # event-specific fields
+  title:
```

Comment:
If we're including `title`, we should include `namespace`.  Also, it seems that we should give this a prefix of `page_` since the `title` is not really part of the edit event.  This is a denormalization of the concepts of `page` and revision`.  If `page_id`, then `page_namespace` and `page_title`.